osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -20,31 +19,40 @@ public class CatcherArea : Container, IKeyBindingHandler<CatchAction>
     {
         public const float CATCHER_SIZE = 106.75f;
 
-        public readonly Catcher MovableCatcher;
+        public Catcher MovableCatcher
```

Comment:
I'm not against this, but I think there should be xmldoc, probably on the `CatcherArea` class explaining that a catcher needs to be set on this property in order for things to work. That said, is there a reason this isn't in the `ctor` args?

Also, shall we rename this to `Catcher`? I'm not sure `MovableCatcher` adds anything any more..